README.md

```diff
@@ -15,6 +15,11 @@
 [![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat&v=3)](https://github.com/RichardLitt/standard-readme)
 [![Makefile Included](https://img.shields.io/badge/Makefile-Supported%20-brightgreen?=flat&logo=probot&v=3)](Makefile)
 <br/>
+
+## Documentation
+
+For in-depth information and guidance, please refer to the [SPV Wallet Documentation]([link here](https://bsvblockchain.gitbook.io/docs)).
+
```

Comment:
As @dorzepowski commented in the other repo:

> It actually should be somewhere below Table Of contents,

Reply:
I would put the sentence at the end of `About` paragraph:

```
More in-depth information and guidance about SPV Wallet Toolbox you can find [here](https://bsvblockchain.gitbook.io/docs).

Reply:
Added sentence into `About` section.

Reply:
Please remove lines 18-22

Reply:
Removed